Source/ARTRealtimePresence.m

```diff
@@ -622,4 +660,315 @@ - (void)failPendingPresence:(ARTStatus *)status {
     }
 }
 
+- (void)broadcast:(ARTPresenceMessage *)pm {
+    [_eventEmitter emit:[ARTEvent newWithPresenceAction:pm.action] with:pm];
+}
+
+- (void)sync {
+    [self sync:nil];
+}
+
+- (void)sync:(ARTCallback)callback {
```

Comment:
We don't have a sync method on `realtime presence` as per [class Presence => protocol 2.0](https://sdk.ably.com/builds/ably/specification/main/features/#TO3l10:~:text=class%20RealtimePresence). Also, `_syncChannelSerial` as a value is not valid anymore. I have removed implementation for `sync` in ably-java. You can check https://github.com/ably/ably-java/blob/0af4d4e465d777462e92845078f0cd966049541b/lib/src/main/java/io/ably/lib/realtime/ChannelBase.java#L371

Reply:
https://github.com/ably/ably-cocoa/pull/1870#discussion_r1510398105

Reply:
I don't think it says what you think it says. It's telling to write a local test where we mock server sent `SYNC` message and inject it client side as a part of test.

Reply:
Duplicate of https://github.com/ably/ably-cocoa/pull/1870/files#r1509076164